### PR TITLE
Authorization: add code examples

### PR DESCRIPTION
**Pull request by alexislefebvre**
I took examples from Symfony's documentation: https://symfony.com/doc/4.4/mercure.html#authorization

### CODE REVIEW

**Comment by dunglas** [comment]
Hi, thanks for your contribution. In my opinion this shouldn't be in the spec itself. The spec shouldn't reference non-standard features such as passing custom headers with `EventSource` and don't have to be exhaustive regarding other use cases covered by other specs such as EventSource (implementers must refer to the linked spec instead).

Would you mind to move these examples in [the FAQ](https://github.com/dunglas/mercure/tree/main/docs/spec/faq.md) instead?

**Comment by alexislefebvre** [comment]
@dunglas hi, I moved the text to the FAQ.

File: docs/spec/faq.md [hunk 1]
```diff
@@ -50,3 +50,24 @@ Updates for the given subscription can then be sent from the GraphQL server to t
...
+
+Cookies are automatically sent by the browsers when opening an `EventSource` connection if the `withCredentials` attribute is set to `true`:
+
+```javascript
+const eventSource = new EventSource(hub, {
+    withCredentials: true
+});
+```
+
+If the client is not a web browser, cookies can't be used.
```
Comment (dunglas) [inline docs/spec/faq.md:64]: This should be removed, most popular HTTP clients including `curl` also support cookies.

File: docs/spec/faq.md [hunk 2]
```diff
@@ -50,3 +50,24 @@ Updates for the given subscription can then be sent from the GraphQL server to t
...
+In order to achieve that, use a [polyfill](https://github.com/Yaffle/EventSource):
```
Comment (dunglas) [inline docs/spec/faq.md:65]: Actually the polyfill is only useful for very old browsers and for cross-domain authorization.

**Review by dunglas — CHANGES_REQUESTED** [review]
We are almost there, thanks!

**Review by dunglas — APPROVED** [review]

**Pull request merged**